server/src/main/scala/com/github/saxypandabear/songrequests/oauth/OauthTokenManager.scala

```diff
@@ -1,17 +1,35 @@
 package com.github.saxypandabear.songrequests.oauth
 
-abstract class OauthTokenManager(clientId: String, clientSecret: String, refreshToken: String, uri: String) {
+/**
+ * A class that handles the OAuth token for Twitch. Note that this does not
+ * need to be generic enough to also handle Spotify authentication, because
```

Comment:
Might need to refactor this when the time comes to rewrite the Spotify lambda code